### ISSUE TITLE
Feature request: Select and drop tiles faster

### ISSUE THREAD

**Issue by maxamine** [christopho/solarus-quest-editor#197]
it would be very useful if the editor allows to select and drop tiles faster using the arrow buttons after selecting a tile. in other words, in map editor, when we select a tile in tileset view in order to put it in the map, we can use the arrows to select tiles around it without having to select each tile using the mouse.
See the screenshots below. 

![1](https://cloud.githubusercontent.com/assets/14140919/16808968/7ec18692-4916-11e6-90eb-bd4d1250a088.png)

![2](https://cloud.githubusercontent.com/assets/14140919/16808979/872d773c-4916-11e6-84ee-2e15f851d396.png)

Feedback welcome

**Comment by Diarandor** [comment]
The idea is good, but the arrow buttons are used to move the scrolling bars, so if this behavior is implemented the old one would have to be changed.

**Comment by oclero** [comment]
You can add a modifier (<kbd>Alt</kbd> or something)

**Comment by Diarandor** [comment]
I agree, using a modifier is probably the best idea (I think it is less annoying than pressing two times the same arrow very fast).

**Comment by maxamine** [comment]
Diarandor; i'm not sur about what you said, actually, if i click on arrows the scrolling bars  doesn't move.
Oclero; what do you mean by ''add a modifier''? it means  using alt+arrows?

**Comment by Diarandor** [comment]
Yes, oclero means "Alt + arrows". If the focus is in the left box (where the tiles are), then the scrolling bars of that part are moved with the arrows. That is why it may be convenient either to add the key modifier; although maybe just changing the current behavior for the new one could be ok.
